code/scripting/api/libs/ui.cpp

```diff
@@ -654,14 +668,102 @@ ADE_FUNC(skipTraining,
 	return ADE_RETURN_NIL;
 }
 
-ADE_FUNC(startBriefingMap,
+// For now any loadout error checking needs to happen in the script to prevent FSO from
+// generating a popup that will not be interactible from Librocket. A later update to this
+// method will introduce return values instead of generating popups and those return values
+// can be used to handle loadout popups on the script side
+ADE_FUNC(commitToMission,
+	l_UserInterface_Brief,
+	nullptr,
+	"Commits to the current mission with current loadout data, and starts the mission. WIP, do not use!",
+	nullptr,
+	"nothing")
```

Comment:
just make it a ``nullptr`` here as well instead of nothing, as you did with the other functions as well